### ISSUE TITLE
False IDE0055 warning (Fix formatting) for extended property patterns

### ISSUE THREAD

**Issue by alrz** [dotnet/roslyn#55328]
```cs
   case BoundDagValueTest { Value.IsBad: false } t: // warns on `:` with no fix
```

**Comment by alrz** [comment]
I think this is resolved by https://github.com/dotnet/roslyn/pull/55646
